src/main/scala/uk/gov/nationalarchives/omega/api/common/AppLogger.scala

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2023 The National Archives
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy of
+ * this software and associated documentation files (the "Software"), to deal in
+ * the Software without restriction, including without limitation the rights to
+ * use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of
+ * the Software, and to permit persons to whom the Software is furnished to do so,
+ * subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
+ * FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR
+ * COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
+ * IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+ * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+package uk.gov.nationalarchives.omega.api.common
+
+import cats.effect.IO
+import org.typelevel.log4cats.{ LoggerFactory, SelfAwareStructuredLogger }
+import org.typelevel.log4cats.slf4j.Slf4jFactory
+
+trait AppLogger {
+  implicit val logging: LoggerFactory[IO] = Slf4jFactory[IO]
+
+  def getAppLoggerFromName(name: String): SelfAwareStructuredLogger[IO] =
+    logging.getLoggerFromName(name)
```

Comment:
I think we can simplify all of this by replacing the above with:
```
  implicit val logger: SelfAwareStructuredLogger[IO] = logging.getLoggerFromName(this.getClass.getSimpleName)
```
We can then use `logger` directly, wherever we need it, and it will have the correct class name. Also because it is implicit we won't need to explicitly pass it around.

Reply:
Done